docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/media-player/index.md

```diff
@@ -53,34 +60,117 @@ If you do not do this the package will still work, but the incremental upserts w
+
+```yml title=dbt_project.yml
+...
+vars:
+  snowplow_media_player:
+    snowplow__atomic_schema: schema_with_snowplow_events
+    snowplow__database: database_with_snowplow_events
+    snowplow__events_table: table_of_snowplow_events
+```
+
+:::info Databricks only
 
-Please refer to the `Quick Start` guide for the Snowplow Web package to make sure you configure the web model appropriately. (e.g. checking the source data or enabling desired contexts).
+Please note that your `target.database` is NULL if using Databricks. In Databricks, schemas and databases are used interchangeably and in the dbt implementation of Databricks therefore we always use the schema value, so adjust your `snowplow__atomic_schema` value if you need to.
 
-### 4. Enable desired contexts
+:::
+
+### 4. Filter your data set
 
-If you have enabled a specific context you will need to enable it in your `dbt_project.yml` file:
+You can specify both `start_date` at which to start processing events, the `app_id`'s to filter for, and the `event_name` value to filter on. By default the `start_date` is set to `2020-01-01`, all `app_id`'s are selected, and all events with the `com.snowplowanalytics.snowplow.media` or the `media_player_event` event name are being surfaced. To change this please add/modify the following in your `dbt_project.yml` file:
 
-```yaml
- dbt_project.yml
+```yml title=dbt_project.yml
 ...
 vars:
   snowplow_media_player:
-    # set to true if the YouTube context schema is enabled
-    snowplow__enable_youtube: true
+    snowplow__start_date: 'yyyy-mm-dd'
+    snowplow__app_id: ['my_app_1','my_app_2']
+    snowplow__media_event_names: ['media_player_event', 'my_custom_media_event']
+```
+### 5. Additional vendor specific configuration
+
+:::info BigQuery Only
+Verify which column your events table is partitioned on. It will likely be partitioned on `collector_tstamp` or `derived_tstamp`. If it is partitioned on `collector_tstamp` you should set `snowplow__derived_tstamp_partitioned` to `false`. This will ensure only the `collector_tstamp` column is used for partition pruning when querying the events table:
+
+```yml title=dbt_project.yml
+...
+vars:
+  snowplow_media_player:
+    snowplow__derived_tstamp_partitioned: false
+```
+
+:::
+
+### 6. Enable desired contexts and configuration
+
+The media player package creates tables that depend on the existence of certain context entities that are tracked by the media plugins in the Snowplow trackers. Depending on which media plugin or tracking implementation you, you will need to enable the relevant contexts in your `dbt_project.yml`.
+
+#### 6a. Using the latest Snowplow Media plugin, Vimeo plugin for JavaScript tracker or iOS/Android trackers
+
```

Comment:
It's not clear to me if these are example or the only possible options of configurations?

Reply:
They are the only possible options at the current time. Have tried to make it more clear (changed the heading and explained in the first paragraph).